openedx/core/djangoapps/catalog/utils.py

```diff
@@ -101,18 +106,63 @@ def get_program_types(user=None):  # pylint: disable=redefined-builtin
         return []
 
 
-def get_programs_data(user=None):
-    """Return the list of Programs after adding the ProgramType Logo Image"""
-
-    programs_list = get_programs(user)
-    program_types = get_program_types(user)
-
-    program_types_lookup_dict = {program_type["name"]: program_type for program_type in program_types}
+def _get_program_instructors(program):
+    """
+    Returns the list of instructor from cached if cache key exists otherwise
+    iterate over the courses and return all the instructors of each course run
+    """
+    cache_key = 'program.instructors.{program_id}'.format(
+        program_id=program.get('uuid')
+    )
+
+    lookup_list = []
+    instructors = cache.get(cache_key) or []
+    if instructors:
+        return instructors
+
+    for queryset in CourseOverview.objects.filter(id__in=_get_all_course_run_keys(program)).values_list(
+            'instructor_info', flat=True):
+        queryset = json.loads(queryset)
```

Comment:
Doesn't `instructor_info` automatically get converted to a dict? Or does the use of values_list prevent that?